instrumentation/ktor-1.0/library/src/main/kotlin/io/opentelemetry/instrumentation/ktor/v1_0/KtorNetServerAttributesExtractor.kt

```diff
@@ -16,15 +15,23 @@ internal class KtorNetServerAttributesExtractor : NetServerAttributesExtractor<A
     return SemanticAttributes.NetTransportValues.IP_TCP
   }
 
-  override fun peerName(request: ApplicationRequest): String {
-    return request.origin.host
+  override fun peerName(request: ApplicationRequest): String? {
+    var remote = request.local.remoteHost
```

Comment:
https://api.ktor.io/ktor-http/io.ktor.http/-request-connection-point/remote-host.html states that it is either client address or host name. What it returns seems to have changed between 1.0 and the latest + there are multiple implementations of `RequestConnectionPoint` interface that don't behave in the same way.